content/en/docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -106,29 +108,74 @@ when the Pod is created, so it is ignored by the scheduler).  Therefore:
  - The DaemonSet controller can make Pods even when the scheduler has not been started, which can help cluster
    bootstrap.
 
-Daemon Pods do respect [taints and tolerations](/docs/concepts/configuration/taint-and-toleration),
-but they are created with `NoExecute` tolerations for the following taints with no `tolerationSeconds`:
 
- - `node.kubernetes.io/not-ready`
- - `node.alpha.kubernetes.io/unreachable`
+### Scheduled by default scheduler
+
+{{< feature-state state="alpha" for-kubernetes-version="1.11"}}
+
+A DaemonSet ensures that all eligible nodes run a copy of a Pod. Normally, the
+node that a Pod runs on is selected by the Kubernetes scheduler. However,
+DaemonSet pods are created and scheduled by the DaemonSet controller instead.
+That introduces the following issues:
+
+ * A DaemonSet cannot respect a Node’s resource changes, such as the node
+   having more free resources after other Pods exit ([#46935](https://github.com/kubernetes/kubernetes/issues/46935)
+ * Inconsistent Pod behavior: Normal Pods waiting to be scheduled are created
+   and in `Pending` state, but DaemonSet pods are not created in `Pending`
+   state. This is confusing to the user.
+ * With preemption, different components, such as DaemonSets and the default
+   scheduler, may make different preemption decisions, which can cause
+   unexpected behaviorss.
```

Comment:
Rephrase it to:

* Pod preemption is handled by default scheduler. When preemption is enabled, the DaemonSet controller will make scheduling decisions without considering pod priority and preemption.